src/frontend.c

```diff
@@ -56,6 +60,32 @@ int main(int argc, char *argv[]) {
 #endif
   logd("Starting frontend\n");
 
+  // init canvas from file
```

Comment:
I'd like to get `stdin` working so you can pipe input from other programs to init a canvas. I'm running into some issues right now though.